### PR TITLE
Verb cleanup

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Removes the vast majority of object verbs, replaced with click combinations that are displayed on examine (Did you know glasses had an adjust verb? If not, you can probably see the first issue with object verbs). Mostly done with this, if I don't finish this today, I will tomorrow.
It's time we damn these to hell, outdated as fuck

List of verbs: (Note, removed just means that the verb itself was removed and wasn't replaced. This doesn't mean that it's behavior is removed as it already had alternatives, only real removed is that just strange verb on AI upload consoles and move to top.)
Eject DNA Scanner - Now an Altclick option
Enter DNA Scanner - Removed
Guardian verbs - Action buttons now
Toggle Pinpointer Mode - Removed
Remove Beaker (Sleeper verb) - Removed, in UI
Enter Sleeper - Removed
Eject ID Card (ID consoles) - Replaced with Alt-click and removes an exploit related to it
Access Computer's Internals (AI upload console) - Removed
Eject Pod (Cryopod) - Removed as you can just move to get out of it, also doesn't work if you're not in it
Enter Pod (Cryopod) - Removed
Rotate Tachyon-doppler Dish - Removed
Rotate Frame (Mass driver) - Alt click
Rotate (Pipe), Flip (Pipe) - Removed
Eject occupant (Recharge station) - Alt click now
Move inside (Recharge station) - Drag and drop now
Rotate Clockwise (Recycler) - Removed
Rotate counterclockwise (Recycler) - Alt-Click
Eject Suit Storage Unit - removed
Hide in Suit Storage Unit - Removed 
Start Washing - AltClick
Climb out - Cannot be triggered in game
Move To Top - Removed as of now due to inconsistency, Denth mentioned having this being scroll wheel based, I like the idea but I'm not sure how we can implement it
Toggle light - AltClick
Alter Beacon's Signal - No longer functions
Switch Verbosity - Removed 
Wipe Tape - AltClick
Spin Chamber - AltClick
Label Disk - Inaccessible
Remove Guest Pass - AltClick
Toggle Paddles - Removed (Exploity)
Remove Bio-chip (Implant reader) - AltClick
Switch Strap Side - AltShiftClick
Switch Gathering Method - AltShiftClick
Empty Content - Inaccessible
Toggle Mister - Removed
Climb structure - Removed 
Toggle Open - Removed
Open/Close - Not functional
Remove Fire Axe - Removed
Toggle Lock - Removed
Activate Electric Chair - AltClick
Rotate (Reflectors) - Removed
Rotate Chair - Removed
Flip table - AltShiftClick
Put table back - AltShiftClick
Rotate Windoor Assembly - Removed
Flip Windoor Assembly - AltShiftClick
Rotate Window Clockwise - Removed
Rotate Window Counter-Clockwise - Removed
Rotate Infrared Laser - AltClick
Hide (Mousetrap) - AltClick
Eject occupant (Cryotube) - Removed and now done automatically
Move Inside - Removed
Adjust Eyewear - AltClick
Toggle Suit Sensors - Removed
Roll Down Jumpsuit - CtrlShiftClick
Remove Accessory - Removed
Holster - Removed
Blob Telepathy - Action Button
Toggle Tank Lid - Altclick
Toggle Tank Light - AltShiftClick
Empty Gibber - AltClick
Card verbs - Action button'd
Change Portal Destination (Contractor Jaunter) - AltClick
Pull - Still exits just not in verb menu
Unfold Chassis & Collapse Chassis - Action Button
Choose Chassis & Choose Speech Verbs - Now in PAI ui
Software interface - Action Button
Slime Verbs - Removed
Eject Photo (Newscaster) - AltClick
Remove clipboard pen - Removed
Eject ID Card (Fax machine) - AltClick
Rename paper - AltClick 
Rename bundle - AltClick
Loose bundle - AltShiftClick
Rename photo - AltClick
Set Photo Focus - AltClick
Print picture - AltShiftClick
Delete picture - CtrlShiftClick
Reset PDA - Now done with a screwdriver
Remove pen - Removed
Remove ID - Removed
Point To - No longer shows up in the verb menu
Rotate (Emitter) - Better AltClick
Rotate Clockwise (PA) - Removed
Rotate Counter-Clockwise - Removed
Toggle Gun Light - Inaccessible
Adjust Tension (bow) - Altclick
Set transfer amount - Removed
Empty Applicator (Mender) - AltClick
Remove Assembly (Beaker) - Screwdriver
Empty Spray Bottle - Altclick
Flip Pipe (Disposals pipes) - AltShiftClick
Rotate Pipe - Removed

## Why It's Good For The Game
A couple of reasons
1. The vast majority of these don't work (such as having unfulfillable activation conditions) or have direct alternatives within the same player area (did you know holster had a verb?), these aren't being maintained and they quite frankly won't be. 
2. Depending on your PC, you may be unable to move while focusing the object verb menu, this is bad for obvious reasons and click combinations are better in general.
3. Players are taught rather early on to examine everything to gain information about it, not right click everything. Most people won't know these features and there's no reason to keep it from them.
4. Fucking hell man these are clunky what do you even want me to say?

Admin UI needs an overhaul, pull up the admin verbs menu and tell me otherwise.

## Testing
So far so good, lots of clicking on items

## Changelog
:cl:
tweak: Most object verbs have been replaced with click combinations, examine the object to view the required combination 
/:cl:

### CODE REVIEW

**Comment by Sirryan2002** [comment]
Can you make a list of the removed verbs? I want to double-check that we're not removing something that has an important (but rare) use.

**Comment by GDNgit** [comment]
> Can you make a list of the removed verbs? I want to double-check that we're not removing something that has an important (but rare) use.

Of course, once I'm done with the PR I'll slap it in there

**Comment by GDNgit** [comment]
This pr now only handles specific datum verbs, admin verbs and pref verbs will come in another pr

File: code/modules/projectiles/guns/projectile/revolver.dm [hunk 1]
```diff
@@ -57,24 +56,16 @@
...
 	if(istype(magazine, /obj/item/ammo_box/magazine/internal/cylinder))
 		var/obj/item/ammo_box/magazine/internal/cylinder/C = magazine
 		C.spin()
 		chamber_round(0)
-		playsound(loc, 'sound/weapons/revolver_spin.ogg', 50, 1)
-		usr.visible_message("[usr] spins [src]'s chamber.", "<span class='notice'>You spin [src]'s chamber.</span>")
-	else
-		verbs -= /obj/item/gun/projectile/revolver/verb/spin
+		playsound(get_turf(user), 'sound/weapons/revolver_spin.ogg', 50, 1)
+		user.visible_message("[usr] spins [src]'s chamber.", "<span class='notice'>You spin [src]'s chamber.</span>")
```
Comment (larentoun) [inline code/modules/projectiles/guns/projectile/revolver.dm:68]: why not the first one is `notice` too?

File: code/modules/reagents/reagent_containers.dm [hunk 2]
```diff
@@ -30,25 +26,20 @@
...
 		return
 
-	if(!usr.Adjacent(src))
-		to_chat(usr, "<span class='warning'>You have moved too far away!</span>")
+	if(!Adjacent(user))
+		to_chat(user, "<span class='warning'>You have moved too far away!</span>")
 		return
-	if(!ishuman(usr) || HAS_TRAIT(usr, TRAIT_HANDS_BLOCKED))
-		to_chat(usr, "<span class='warning'>You can't use your hands!</span>")
+	if(!ishuman(user) || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
```
Comment (larentoun) [inline code/modules/reagents/reagent_containers.dm:32]: Not sure if `ishuman()` is needed here

**Review by larentoun — CHANGES_REQUESTED** [review]
This is what I noticed in 30 minutes of quick glance

File: code/game/machinery/rechargestation.dm [hunk 3]
```diff
@@ -170,40 +170,39 @@
...
 	change_power_mode(IDLE_POWER_USE)
-	return
+
+/obj/machinery/recharge_station/AltClick(mob/user)
+	if(user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user))
+		return
+	go_out()
 
 /obj/machinery/recharge_station/force_eject_occupant(mob/target)
 	go_out()
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/game/machinery/rechargestation.dm:180]: Yeah this is a helper for admin procs

File: code/game/machinery/washing_machine.dm [hunk 4]
```diff
@@ -21,16 +21,18 @@
...
-	if(!isliving(usr)) //ew ew ew usr, but it's the only way to check.
+/obj/machinery/washing_machine/AltClick(mob/user)
+	if(user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user))
 		return
+	start(user)
 
-	if( state != 4 )
-		to_chat(usr, "The washing machine cannot run in this state.")
+/obj/machinery/washing_machine/proc/start(mob/user)
+	if(state != 4)
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/game/machinery/washing_machine.dm:34]: The washing machine is a little shit already and I'm touching as little of it as possible

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 5]
```diff
@@ -229,25 +229,13 @@
 
 /obj/structure/closet/crate/secure/AltClick(mob/user)
 	if(Adjacent(user) && !opened)
-		verb_togglelock()
+		if(HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || user.stat ) // Don't use it if you're not able to! Checks for stuns, ghost and restrain
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/game/objects/structures/crates_lockers/crates.dm:232]: adjacent two lines above that

File: code/game/objects/structures/reflector.dm [hunk 6]
```diff
@@ -100,30 +100,14 @@
...
+	if(user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user))
 		return
-	if(!in_range(src, user))
+
+	if(anchored)
+		to_chat(usr, "<span class='warning'>You cannot rotate [src] right now. It is fastened to the floor!</span>")
 		return
-	else
-		rotate()
+	dir = turn(dir, 90)
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/game/objects/structures/reflector.dm:110]: yeah

File: code/modules/clothing/clothing.dm [hunk 7]
```diff
@@ -225,22 +225,18 @@
...
-	var/mob/living/carbon/human/user = usr
-	if(!istype(user))
-		return
-	if(user.incapacitated()) //Dead spessmen adjust no glasses. Resting/buckled ones do, though
+/obj/item/clothing/glasses/examine(mob/user)
+	. = ..()
+	. += "<span class='info'>You can <b>Alt-Click</b> [src] to adjust if it fits over or under your mask.</span>"
+
+/obj/item/clothing/glasses/AltClick(mob/living/carbon/human/user)
+	if(user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user) || !istype(user))
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/modules/clothing/clothing.dm:233]: doesn't really matter

File: code/modules/reagents/reagent_containers.dm (hunk 2 above)
Comment (GDNgit) [reply code/modules/reagents/reagent_containers.dm:32]: input menu, sleeps

File: code/modules/mob/mob.dm [hunk 8]
```diff
@@ -757,6 +757,8 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 /mob
 	var/newPlayerType = /mob/new_player
 
+// Nobody in their right mind will have this enabled on the production server, uncomment if you want this for some reason
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/modules/mob/mob.dm:760]: right now you ALWAYS get this verb in the ooc tab so it's just kinda silly

**Review by Sirryan2002 — APPROVED** [review]
lets get this through, what's the worst that could happen. If no one notices these being removed then they probably needed to go anyway.

**Review by S34NW — APPROVED** [review]
Deconflict for merge

**Comment by Dafrek** [comment]
> lets get this through, what's the worst that could happen. If no one notices these being removed then they probably needed to go anyway.

The most noticeable part for me is not being able to toggle suit sensors the old way and pointing at objects that you try to pick up because the bar is one higher than it used to be.

**Pull request merged**